src/WireMock.Net/Settings/WireMockServerSettingsParser.cs

```diff
@@ -145,7 +177,33 @@ public static bool TryParseArguments(string[] args, [NotNullWhen(true)] out Wire
         {
             settings.CertificateSettings = certificateSettings;
         }
+    }
 
-        return true;
+    private static void TryParseWebProxyAddressSettings(WireMockServerSettings settings, SimpleCommandLineParser parser)

```

Comment:
pass here the `ProxyAndRecordSettings` instead of `WireMockServerSettings`

Reply:
Good idea, updated